From 92d10d5d6882c3db6695a8fff83c88fbaaa27a33 Mon Sep 17 00:00:00 2001
From: "Eric S. Raymond" <esr@thyrsus.com>
Date: Thu, 23 Aug 2012 12:46:03 -0400
Subject: Fix malformed list syntax.

TP without a body shouldn't really be used for an item list. Fixing this
makes structural translation to DocBook possible.

Signed-off-by: Eric S. Raymond <esr@thyrsus.com>

diff --git a/man/s3.man b/man/s3.man
index 9109c2a..732e4f3 100644
--- man/s3.man
+++ man/s3.man
@@ -50,9 +50,9 @@ driver supports PCI video cards based on the following S3 chips:
 
 .PP
 Also driver supports the following RAMDACs:
-.TP 12
+.IP 12
 .B IBM 524, IBM 524A, IBM 526, IBM 526DB
-.TP 12
+.IP 12
 .B TI ViewPoint 3025 
 
 .SH DESCRIPTION
-- 
cgit v0.10.2

